history-of-the-quakers-v3/original/29-book-6-chapter-9.adoc

```diff
@@ -157,8 +171,13 @@ William Penn placed himself in the balcony, and thence preached to the people;
 but in the midst of his declaration he was interrupted
 by officers from the mayor with the following warrant:
 
+[.embedded-content-document.legal]
+--
+
+[.signed-section-context-open]
 Wells City and Borough.
```

Comment:
Wasn't sure if this should be above the [.letter-heading] or not, or if I should move it around or not even label it anything.